### PR TITLE
NextJS setup

### PR DESCRIPTION
**Pull request by its-tayo**
NextJS setup with typography, colors and commit hooks

### CODE REVIEW

File: TODO [hunk 1]
```diff
@@ -1,3 +1,8 @@
 
 Todo:
```
Comment (tomide1) [inline TODO]: Let's keep track of todo items on Trello please

Comment (its-tayo) [reply TODO]: I don't want to put those on Trello, I'll add untrack and the file to gitignore

File: components/Forms/BrandSignUpForm/index.jsx [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+const BrandSignUpForm = ({ onSubmit }) => {
+  const validationSchema = object().shape({
+    firstName: string().required(),
+    lastName: string().required(),
+    brandName: string().required(),
+    email: string()
+      .email()
+      .required(),
+    phoneNumber: string().required(),
+    password: string()
```
Comment (tomide1) [inline components/Forms/BrandSignUpForm/index.jsx]: We should agree on a schema for valid password.

File: pages/brands/sign-in/index.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    <>
+      <SEO title="Title" description="description" />
+      <Auth>
+        <p className="text-center font-medium md:hidden">Sign in to Scizzor</p>
+        <h3 className="hidden mb-3 md:block">Sign In to Scizzor</h3>
+        <BrandSignInForm onSubmit={handleSubmit} />
+        <p className="mt-5">Forgot your password?</p>
+        <p>
+          Don't have an account?{' '}
+          <span className="text-primary">Create one here</span>
```
Comment (tomide1) [inline pages/brands/sign-in/index.jsx]: This should route to brands/register

File: TODO (hunk 1 above)
Comment (tomide1) [reply TODO]: That's fine

File: pages/brands/sign-in/index.jsx (hunk 3 above)
Comment (tomide1) [reply pages/brands/sign-in/index.jsx]: We'll revisit once we add the api endpoints to the app

**Pull request merged**